### ISSUE TITLE
Pantheon Drupal 8 SSL error

### ISSUE THREAD

**Issue by rlahoda** [lando/lando#1129]
Feature Requests and Bug Reports
--------------------------------

Please complete the following steps.

**The more steps you complete the more likely we are to address your issue!**

1.  Update to the [latest version](https://github.com/lando/lando/releases) of Lando and make sure the issue isn't fixed already.
2.  Please review the [current list of issues](https://github.com/lando/lando/issues) to make sure you are not submitting a duplicate.
3.  Fill out either the `BUG REPORT` or `FEATURE REQUEST` below.

Bug Report
==========

**Tell us about your setup**

What is your lando version and operating system? **(note that the older your version the less likely we are to reply)**

Lando v3.0.0-beta.47 running on Macbook Pro running OSX 10.11.6, Docker Version 18.06.0-ce-mac70 (26399)

My `.lando.yml` file contents

```yml
name: heritage-insurance-advisors
recipe: pantheon
config:
  framework: drupal8
  site: heritage-insurance-advisors
  id: ...
```

It is also helpful to include other relevant config files. For example, include `pantheon.yml` for a `pantheon` recipe if applicable. Include `php.ini` or `mysql.cnf` if you are overriding our defualt config.

Pantheon.yml:

```yml
# Put overrides to your pantheon.upstream.yml file here.
# For more information, see: https://pantheon.io/docs/pantheon-yml/
api_version: 1
```

**Tell us about the command you were running**

Paste in a series of commands that caused the bug.
All I did was run `lando start`

Here are the contents of what came up after that:
```
Starting landoproxyhyperion5000gandalfedition_proxy_1 ... done
Creating heritageinsuranceadvisors_database_1 ... done
Creating heritageinsuranceadvisors_index_1    ... done
Creating heritageinsuranceadvisors_cache_1    ... done
Creating heritageinsuranceadvisors_appserver_1 ... done
Creating heritageinsuranceadvisors_nginx_1     ... done
Creating heritageinsuranceadvisors_edge_1      ... done
Creating heritageinsuranceadvisors_edge_ssl_1  ... done
Waiting until edge_ssl service is ready...
Waiting until edge service is ready...
Waiting until nginx service is ready...
Waiting until appserver service is ready...
Waiting until database service is ready...
Waiting until cache service is ready...
Waiting until index service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Waiting until edge_ssl service is ready...
Pantheon pre-run scripting
Setting up client key /lando/keys/pantheon/heritageinsuranceadvisors/cert.pem
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   599    0   599    0     0    801      0 --:--:-- --:--:-- --:--:--   801
100 4482k  100 4482k    0     0  1807k      0  0:00:02  0:00:02 --:--:-- 5141k
 PHP configuration      :
 PHP OS                 :  Linux
 Drush script           :  /tmp/drush.phar
 Drush version          :  8.1.16
 Drush temp directory   :  /tmp
 Drush configuration    :
 Drush alias files      :

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  713k  100  713k    0     0   384k      0  0:00:01  0:00:01 --:--:--  384k
 [notice] Logging in via machine token.

BOOMSHAKALAKA!!!

Your app has started up correctly.
Here are some vitals:

 NAME           heritageinsuranceadvisors
 LOCATION       /Users/rob/web_projects/client-heritage-insurance-advisors/local
 SERVICES       appserver, nginx, database, cache, edge_ssl, edge, index
 NGINX URLS     https://localhost:32788
                http://localhost:32789
 EDGE URLS      http://localhost:32790
                http://heritageinsuranceadvisors.lndo.site:8000
                https://heritageinsuranceadvisors.lndo.site:444
 EDGE_SSL URLS  https://localhost:32791
```
The last two urls `https://heritageinsuranceadvisors.lndo.site:444` and `https://localhost:32791` are both in red, the others are in green

**Tell us about the error you got**

When I go to the `.lndo.site:8000` url I get the following error in my browser:
```
An error occurred during a connection to heritageinsuranceadvisors.lndo.site:8000. SSL received a record that exceeded the maximum permissible length. Error code: SSL_ERROR_RX_RECORD_TOO_LONG 
```
If I take the :`:8000` off it takes me to an error 400 page

If I use the `.lndo.site:444` url, I am prompted for a security exception for the page but when the exception is added the only contents that show up on the page are the message: `The provided host name is not valid for this server.`

 I tried rebuilding the site to see if that would solve it, but it didn't. I saw on a different issue a suggestion to run `docker system prune` then try `lando start` again, but that didn't work either. I even started over with a new `lando init pantheon` in a new directory, pulled the db and code and still get the issue. I am completely unable to view anything on the site as a result of this.

**Comment by rlahoda** [comment]
Update: I tried reinstalling beta 46 and Docker wouldn't work anymore, so I went ahead and reinstalled beta 47. After the installation everything started to work. Interestingly, the URL's listed no longer had the port #'s on them so instead of `http://heritageinsuranceadvisors.lndo.site:8000` it now is just `http://heritageinsuranceadvisors.lndo.site`. This might be because of something changing with Docker or something, not sure. But it's working for now so that's what matters.
